controllers/users/register_v2.js

```diff
@@ -28,47 +28,87 @@ const registerV2 = async (req, res) => {
   const {data} = req.body;
   const {users, follows, local_community, topics} = data;
 
+  const checkUser = await User.findOne({
+    where: {
+      human_id: users.human_id
+    }
+  });
+
   let insertedObject = {};
+  let isUnverifiedUser = checkUser && checkUser.verified_status === 'UNVERIFIED';
 
-  /**
-   * Inserting data to Postgres DB
-   */
-  try {
-    insertedObject = await sequelize.transaction(async (t) => {
-      const user = await UsersFunction.register(User, users, t);
-      const anonymousUser = await UsersFunction.registerAnonymous(User, user?.user_id, t);
-
-      const topicRegistered = await TopicFunction.findAllByTopicIds(Topics, topics, t, true);
-      const locations = await LocationFunction.getAllLocationByIds(Locations, local_community, t);
-
-      return {
-        user,
-        anonymousUser,
-        topics: topicRegistered,
-        locations
-      };
-    });
-  } catch (e) {
-    console.log('error on sql transaction', e);
-    return res.status(500).json({
-      status: 'error on sql transaction',
-      code: 500,
-      message: e
-    });
+  if (isUnverifiedUser) {
+    let updatedUser;
+    try {
+      await checkUser.update(users);
+      updatedUser = await checkUser.save();
```

Comment:
verified_status has not changed from UNVERIFIED to VERIFIED if I register using unverified user.
Please fix

Reply:
baik mas